tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -240,6 +240,7 @@ THIRD_PARTY_DOWNLOADS :=
 $(eval $(call add_third_party_download,$(GEMMLOWP_URL),$(GEMMLOWP_MD5),gemmlowp,))
 $(eval $(call add_third_party_download,$(FLATBUFFERS_URL),$(FLATBUFFERS_MD5),flatbuffers,))
 $(eval $(call add_third_party_download,$(RUY_URL),$(RUY_MD5),ruy,))
+$(eval $(call add_third_party_download,$(PERSON_MODEL_URL),$(PERSON_MODEL_MD5),person_model_grayscale,))
```

Comment:
while this would remove the warnings, it would mean that person_model_grayscale is downloaded irrespective of the build target and platform, which is something that we currently want to avoid. The warnings are not great, and a different solution that allows for selective downloads without warnings would be ideal.